src/main/java/com/efficio/fieldbook/web/fieldmap/controller/FieldmapController.java

```diff
@@ -661,6 +662,37 @@ public Map<String, String> getFieldLocations() {
 		return result;
 	}
 
+	/**
+	 * Delete field map.
+	 */
+	@ResponseBody
+	@RequestMapping(value = "/deletion", method = RequestMethod.POST,
+		produces = "application/json; charset=utf-8")
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES','MANAGE_STUDIES')")
```

Comment:
@cheryleigh we are not using PreAuthorize in fieldbook I think it is here because you copied from BMSAPI, right?

Reply:
@clarysabel yup, I didn't notice, thank you!